src/woo_search/search_index/tests/factories.py

```diff
@@ -79,9 +87,23 @@ def informatie_categorieen(
             self["informatie_categorieen"] = extracted
         else:
             size = kwargs.pop("size", 1)
-            categories = InformationCategoryFactory.create_batch(size, **kwargs)
+            categories = NestedInformationCategoryFactory.create_batch(size, **kwargs)
             self["informatie_categorieen"] = categories
 
+    @factory.post_generation
+    def onderwerpen(
+        self: dict[str, Any],  # pyright: ignore[reportGeneralTypeIssues]
+        create,
+        extracted,
+        **kwargs,
+    ):
+        if extracted:
+            self["onderwerpen"] = extracted
+        else:
+            size = kwargs.pop("size", 1)
+            topics = NestedTopicFactory.create_batch(size, **kwargs)
+            self["onderwerpen"] = topics
```

Comment:
I'd drop this. There's going to be more data not related to a topic than data that is related to a topic.

Reply:
I will change it to an empty list instead, this makes writing tests easier. Because the index tasks will expect the value to be given (even when empty). If we don't want to do this then we need to divert from how we have been setting up these fields so I rather not go that route.